chattore/src/main/kotlin/ChattOREConfig.kt

```diff
@@ -0,0 +1,23 @@
+package org.openredstone.chattore
+
+import org.openredstone.chattore.feature.DiscordConfig
```

Comment:
`DiscordConfig` being defined in a feature feels kinda weird, given the rest of the config is defined here.

Reply:
It's in a weird state right now yea :/. The end goal though is that the config of each feature is defined together with the feature itself and perhaps a secondary goal that there would be no ChattOREConfig that ties them together (rather just load them into each feature's config data class directly from a subsection of the config). I will continue to work on this though.